### PR TITLE
rename PluginStepBlobDoesNotExist->PluginStepBlobEtcdBlobExists

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
NONE
```

### CODE REVIEW

File: pkg/api/plugin.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ const (
 	PluginStepInitialize                          PluginStep = "Initialize"
 	PluginStepInitializeUpdateBlob                PluginStep = "InitializeUpdateBlob"
 	PluginStepResetUpdateBlob                     PluginStep = "ResetUpdateBlob"
-	PluginStepBlobDoesNotExist                    PluginStep = "BlobDoesNotExist"
+	PluginStepBlobEtcdBlobExists                  PluginStep = "BlobDoesNotExist"
```
Comment (thekad) [inline pkg/api/plugin.go]: you are not changing the value, so the value and the step say 2 different things

Comment (jim-minter) [reply pkg/api/plugin.go]: fail by me - too much hurry.  Thanks.

**Pull request merged**